src/VisualStudio/Core/Test/Completion/VisualBasicCompletionSnippetNoteTests.vb

```diff
@@ -18,17 +18,23 @@ Class Goo
     $$
 End Class]]></document>
 
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
-        Public Async Function ColonDoesntTriggerSnippetInTupleLiteral() As Task
-            Using state = CreateVisualBasicSnippetExpansionNoteTestState(_markup, "Interface")
+        Private Shared Function GetAllCompletions() As IEnumerable(Of Object())
```

Comment:
:bulb: Make these read-only properties named `AllCompletions`

Reply:
AllCompletionImplementations

---
In reply to: [208790208](https://github.com/dotnet/roslyn/pull/29176#discussion_r208790208) [](ancestors = 208790208)